ocs_ci/ocs/must_gather/must_gather.py

```diff
@@ -44,10 +46,29 @@ def collect_must_gather(self):
         """
         Collect ocs_must_gather and copy the logs to a temporary folder.
 
+        Verify the output of mg command does not contain "cannot stat 'jq'"
+        https://bugzilla.redhat.com/show_bug.cgi?id=1974959
+
         """
-        temp_folder = tempfile.mkdtemp()
-        collect_ocs_logs(dir_name=temp_folder, ocp=False)
-        self.root = temp_folder + "_ocs_logs"
+        self.root = tempfile.mkdtemp()
+        latest_tag = ocsci_config.REPORTING.get(
+            "ocs_must_gather_latest_tag",
+            ocsci_config.REPORTING.get(
+                "default_ocs_must_gather_latest_tag",
+                ocsci_config.DEPLOYMENT["default_latest_tag"],
+            ),
+        )
+        ocs_must_gather_image = ocsci_config.REPORTING["ocs_must_gather_image"]
+        ocs_must_gather_image_and_tag = f"{ocs_must_gather_image}:{latest_tag}"
+        if ocsci_config.DEPLOYMENT.get("disconnected"):
+            ocs_must_gather_image_and_tag = mirror_image(ocs_must_gather_image_and_tag)
+        cmd = f"adm must-gather --image={ocs_must_gather_image_and_tag} --dest-dir={self.root}"
```

Comment:
collect_ocs_logs() handles all these code, can you please let know the reason why the collect_ocs_logs is removed here?

Reply:
I want to verify that "cannot stat 'jq'" string is not included in the output of mg cmd

Reply:
@petr-balogh Can I add this code to `collect_ocs_logs` function?
```
        output = occli.exec_oc_cmd(command=cmd, out_yaml_format=False, timeout=1500)
        if "cannot stat 'jq'" in output:
            raise ValueError(
                f"must-gather fails in an disconnected environment\n{output}"
            )
```

Reply:
Not sure if it can have some effect on existing execution. If not I think it can be added. 
Don't you want to add some specific exception for this instead of use general ValueError?

As you are referring to disconnected env in exception message, this check should be done only when it's disconnected deployment.